bedrock/firefox/redirects.py

```diff
@@ -629,4 +629,11 @@ def firefox_channel(*args, **kwargs):
 
     # issue 7670
     redirect(r'^/firefox/fights-for-you/?', 'firefox'),
+
+    # issue #7424
+    redirect(r'^/firefox/content-blocking/start/', 'https://support.mozilla.org/'),
+
+    # issue #7424
+    redirect(r'^/firefox/tracking-protection/start/', 'https://support.mozilla.org/'),
```

Comment:
The URL to redirect to should be `https://support.mozilla.org/kb/content-blocking` as per the issue description.